docs/updating/4_update_2.2.md

```diff
@@ -89,6 +89,10 @@ You can remove the bundle after the migration is complete.
 The command will migrate Landing Pages created in eZ Platform 1.x, 2.0 and 2.1 to new Pages.
 The operation is transactional and will roll back in case of errors.
 
+The Page Builder in 2.2 does not offer all blocks that Landing Page editor did. The removed blocks include Schedule and Form blocks.
```

Comment:
These exists on 2.5 right? So if possible to change this to approach migration on 2.5 version it won't need to be mentioned or? (or do we need to improve migration for these blocks maybe?)

Reply:
They exist on 2.5, yes. But it is still not possible to migrate to them if running migration script on 2.5. I also found that Tag block has not been added to pagebuilder.. So I'll add info about this one too